mvpa2/datasets/miscfx.py

```diff
@@ -18,6 +18,8 @@
 
 import numpy as np
 
+from tabulate import tabulate
```

Comment:
this introduces additional external dependency (tabulate).  If sticking to it we would need to add conditioning on it being available, add its check within mvpa2.base.externals, etc

alternatively, we have similar one `table2string` but probably it is not as versatile and would require additional massaging of data before giving into it